### PR TITLE
web tests

### PR DESCRIPTION
**Pull request by francescobrenci-AR**
## Describe your changes

Added more tests for utility functions

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/lib/__tests__/libraries.test.js [hunk 1]
```diff
@@ -1,6 +1,50 @@
...
+				throw error;
+			};
+
+			expect(asyncRoute(route)).rejects.toThrowError(error);
+		});
+
+		it('should throw error if route returns rejected promise', () => {
+			const error = new Error('some error');
+
+			// @ts-ignore
```
Comment (lutamKC) [inline apps/web/src/server/lib/__tests__/libraries.test.js:39]: do you still need this ts-ignore?

File: apps/web/.eslintrc.cjs [hunk 2]
```diff
@@ -6,7 +6,10 @@ module.exports = {
 			rules: {
 				// allow devdependencies
 				// https://github.com/weiran-zsd/eslint-plugin-node/blob/cb4f099ae928d627abfc635966622cf4266a7b1c/docs/rules/no-unpublished-import.md
-				'n/no-unpublished-import': 'off'
+				'n/no-unpublished-import': 'off',
+				'jest/valid-expect': 'off',
+				'no-unused-vars': 'off'
```
Comment (lutamKC) [inline apps/web/.eslintrc.cjs:11]: no unused vars could be useful to clean up the code actually

**Review by lutamKC — DISMISSED** [review]

File: apps/web/.eslintrc.cjs (hunk 2 above)
Comment (francescobrenci-AR) [reply apps/web/.eslintrc.cjs]: fair game - put it back, although was only for tests and infrastructure code

File: apps/web/src/server/lib/__tests__/libraries.test.js (hunk 1 above)
Comment (francescobrenci-AR) [reply apps/web/src/server/lib/__tests__/libraries.test.js]: removed

**Review by lutamKC — APPROVED** [review]

**Pull request merged**